buildpacks/buildpacks-v3.yaml

```
@@ -0,0 +1,102 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: buildpacks-v3
+spec:
+  inputs:
+    params:
+    - name: IMAGE
+      description: The image you wish to create. For example, "repo/example", or "example.com/repo/image".
+    - name: RUN_IMAGE
+      description: The run image buildpacks will use as the base for IMAGE.
+      default: packs/run:rc
+    - name: BUILDER_IMAGE
+      description: The builder image (must include v3 lifecycle and compatible buildpacks).
+      default: heroku/buildpacks
```

```suggestion
      default: packs/samples:v3alpha2
```

Align the task yaml with readme or remove the default altogether?

The most recent release of the [pack cli](https://github.com/buildpack/pack) does not include a default builder.